Library/Homebrew/cmd/help.rb

```diff
@@ -91,7 +91,8 @@ def command_help(path)
         line.slice(2..-1).
           sub(/^  \* /, "#{Tty.highlight}brew#{Tty.reset} ").
           gsub(/`(.*?)`/, "#{Tty.highlight}\\1#{Tty.reset}").
-          gsub(/<(.*?)>/, "#{Tty.em}\\1#{Tty.reset}")
+          gsub(/<(.*?)>/, "#{Tty.em}\\1#{Tty.reset}").
+          gsub("@hide_from_man_page", "")
```

Comment:
This creates a spurious empty line in the `brew help` output, because the magic token is removed, but not the line that contains the magic token. Can you modify the filtering to instead drop the entire line that contains this token? Maybe even make the matching a bit stricter and only accept the token if it is the only thing in a documentation line.